tardis/tardis_portal/download.py

```diff
@@ -532,7 +532,7 @@ def get_datafile(dfid):
         expid = request.POST['expid']
         experiment = Experiment.objects.get(id=expid)
     except (KeyError, Experiment.DoesNotExist):
-        experiment = iter(df_set).next().dataset.get_first_experiment()
+        experiment = iter(df_set).__next__().dataset.get_first_experiment()
```

Comment:
Can you please use the built-in `next()` function, instead of the `__next__()` method?  i.e. `next(iter(df_set)).dataset.get_first_experiment()`.  They we get Python 2 compatibility so this fix can be backported to the v4.1 branch.  And using the built-in `next()` function is the recommended way to do this - see https://www.python.org/dev/peps/pep-3114/#transition-plan which says "x.next() will become next(x)".